### DISCUSSION TITLE
variable/term to include main prompt.

### DISCUSSION THREAD

**Discussion by Jonathanese** [Bing-su/adetailer#400] [Ideas]
Basically instead of choosing the original prompt OR a new prompt, basically just append the original prompt. But as a bonus, instead of adding the terms to the beginning or the end, have a term you can enter which will "paste" the original prompt at that location. Perhaps with a weight assigned to it or something.

So the idea would look something like this.

```
Main Prompt: "Keanu Reeves fighting a bear"
Negative Prompt: "Matrix, Neo, Movie"

ADetailer face prompt: "Detailed closeup, <prompt:0.25>, facial hair"
ADetailer face negative prompt: "Makeup, <neg_prompt:0.5>, panda"
```

The advantage here is that you can basically toss all your common detail information for each adetailer prompt, but it'll still "remember" the original subject. And based on where you position it, you can emphasize the base prompt more or less. So you might have the subject have high priority on the face, but almost no priority on the hands.

So you might do some of the following:

```
Hand+: "closeup shot of hand, detailed, <prompt:0.1>
Hand-: "crossed fingers, wrinkly, <neg_prompt:1>

Face+: "Closeup portrait, <prompt:1>, detailed, flawed skin"
Face-: "painted, picasso, pose, <neg_prompt:0.5>"
```

From there, you can make whatever prompts you want, and the prompts will be tweaked to optimize for each feature.

**Comment by Zotikus1001** [comment]
This would be amazing.
